src/audiovideocontroller/DefaultAudioVideoController.ts

```diff
@@ -425,8 +438,10 @@ export default class DefaultAudioVideoController implements AudioVideoController
     });
   }
 
-  update(): boolean {
+  update(videoSubscribeContext: VideoSubscribeContext): boolean {
+    const nextVideoSubscribeContext = videoSubscribeContext.clone();
```

Comment:
nit: how about removing this line and just assigning it to `this.meetingSessionContext`? Creating the *next* and immediately assigning it to *current* don't explain much.